### ISSUE TITLE
Callback should take session parameter

### ISSUE THREAD

**Issue by mirosval** [3tty0n/gatling-thrift#10]
thank you for all the effort on this library!

I would like to use the `Session` parameter like this:

```scala
def deleteImage(session: Session): Future[Boolean] = {
  thriftClient.deleteImage(session("i").as[String])
}

...

val deleteImages = scenario("DeleteImages")
    .feed(images.shuffle.circular)
    .exec(deleteImage)
```

This is currently not possible because the implicit callback builder looks like this:

```scala
implicit def callbackToThriftActionBuilder[A](
      callback: => Future[A]
  ): ThriftActionBuilder[A] = ThriftActionBuilder(callback)
```

so it does not take a `session` parameter.

**Comment by 3tty0n** [comment]
@mirosval Sorry for my late reply and thank you for your reporting!
I created #11 to support `session` and this will be included in `0.4.0`, the next major version.

**Comment by 3tty0n** [comment]
I just released `0.4.0` and `0.4.1` to maven central. 
Thank you for your contributing!
